packages/identity-sdk/src/wagmi-sdk.ts

```diff
@@ -0,0 +1,108 @@
+import { useCallback } from "react";
+import { usePublicClient, useWalletClient } from "wagmi";
+import { Address, PublicClient } from "viem";
+import {
+  initializeIdentityContract,
+  isWhitelisted,
+  generateFVLink,
+  getIdentityExpiryData,
+} from "./viem-sdk";
+
+interface IdentitySDK {
+  checkIsWhitelisted: (account: Address) => Promise<boolean>;
+  generateFVLink: (
+    callbackUrl?: string,
+    popupMode?: boolean,
+    chainId?: number,
+  ) => Promise<string>;
+  getIdentityExpiry: (account: Address) => Promise<IdentityExpiry | undefined>;
+}
+
+interface IdentityExpiry {
+  authPeriod: bigint;
+  expiryTimestamp: bigint;
+  formattedExpiryTimestamp?: string;
+  lastAuthenticated: bigint;
+}
+
+export const useIdentitySDK = (contractAddress: Address): IdentitySDK => {
```

Comment:
contractAddress can be as previous prod,dev,staging 0xstring

Reply:
done

Reply:
That also means for the viem-sdk
a dev using viem sdk should also simply specify dev/prod

Reply:
done as part of the refactor to a class